From 3e68615dd49a0f6f1e3494dde012bbafd98218bd Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Tue, 30 Aug 2022 14:55:51 +0300
Subject: [PATCH] [SW-102303] gaudi3: update fw version fuse offset

As per latest fuse spec, firmware version base is shifted to row 61,
so update accordingly.

Change-Id: Ied4998fa0629c2428eb3d299ab92b74061004681
---

diff --git a/arch/gaudi3/arc/dev_efuse.h b/arch/gaudi3/arc/dev_efuse.h
index cf4c058..32b1378 100644
--- a/arch/gaudi3/arc/dev_efuse.h
+++ b/arch/gaudi3/arc/dev_efuse.h
@@ -9,7 +9,7 @@
 /* Unused for gaudi3. Maintained for common code compilation */
 #define EFUSE_KEY_HASH_OFFSET				0x0
 #define EFUSE_SYSCFG2_OFFSET				0x3B
-#define EFUSE_SECURITY_FW_VERSION_OFFSET		0x3C
+#define EFUSE_SECURITY_FW_VERSION_OFFSET		0x3D
 
 #define EFUSE_SYSCFG_REVOKE_KEY_0			BIT(0)
 #define EFUSE_SYSCFG_REVOKE_KEY_4			BIT(4)
